index.js

```diff
@@ -33,6 +33,10 @@ function recurse(nodes, test) {
           memo = memo.concat(recurse([node], test));
         }
       }).filter(Boolean), test, memo));
+      // If els contains another conditional expression, continue recursion
+      if ( node.els.type = 'ConditionalExpression' ) {
+        memo.push.apply(memo, recurse([node.els], test));
+      }
```

Comment:
I'm not 100% clear on the structure of the tree; it's possible we would *always* want to recurse down to `node.els` and `node.elsif`, since otherwise their `node.nodes` will not be parsed. If there is no situation where these would not have `type` of `'ConditionalExpression'`, I can remove this conditional as unnecessary